tests/ha/check_cluster_integrity.pm

```diff
@@ -17,6 +17,7 @@ use utils 'systemctl';
 
 sub run {
     select_console 'root-console';
+    sleep 120;
```

Comment:
It is generally not good to simply wait, instead of waiting for a condition. In the best scenario, a `sleep` will wait longer than needed, artificially lengthening the time a worker is busy on the test, and in the worst scenario it will not be enough in slower architectures.

I suggest to try `wait_until_resources_started` which will wait until 120 seconds while checking the cluster state and finish before the 120 seconds if all resources are started.